cborg/src/Codec/CBOR/Decoding.hs

```diff
@@ -1047,7 +1125,7 @@ peekByteSpan = Decoder (\k -> return (PeekByteSpan k))
 --
 -- > openByteSpan
 -- > x <- decode
--- > !after  <- peekByteSpan
+-- > bytes <- peekByteSpan
```

Comment:
you removed bang here, but not in the implementation.

Reply:
Thanks. I think the bang there is unnecessary and it's better to force the call of `peekMarkedByteSpan` as it is passed to the continuation. I'll change that.